YamlFile.js

```diff
@@ -142,6 +142,11 @@ var badEndPunct = {
 YamlFile.prototype._isTranslatable = function(resource) {
     var locale = new Locale(this.getLocale());
 
+    var yamlSettings = this.project.settings.yaml;
+
+    // if extra config exists in yaml settings (project.json) and checkNonTranslatable is set to false then check will be skipped
+    if ((yamlSettings.config) && !(yamlSettings.config.checkNonTranslatable)) return true;
```

Comment:
Actually, I would reverse the sense of it. Currently, we check if lines are translatable, and we want to turn that check off with this setting.

`if (yamlSettings && yamlSettings.checkTranslatability === false) return true;`

Reply:
right, fixed.